app/src/main/res/values/strings.xml

```diff
@@ -84,13 +84,14 @@
     <string name="iris_scan_left_title">Iris scan - Left eye</string>
     <string name="iris_scan_right_title">Iris scan - Right eye</string>
     <string name="participant_matching_title">Child matching</string>
+    <string name="visits_overview_client_mothers_name_column_label">Mother's Name</string>
     <string name="participant_flow_child_number_title">Child Number</string>
     <string name="participant_flow_participant_id_title">Child ID</string>
     <string name="participant_flow_participant_id_hint">Scan Child ID QR Code</string>
     <string name="participant_flow_participant_id_scanned">The format of the ID is not supported</string>
     <string name="participant_flow_participant_id_description">Please scan the Child vaccination card with a QR-Code or manually enter the Child ID number:</string>
-    <string name="participant_flow_participant_id_label_manual_entry">Manual ID Entry</string>
-    <string name="participant_flow_participant_id_input_hint">#1234567890</string>
+    <string name="participant_flow_participant_id_label_manual_entry">Manual Enter ID or Search by mothers Name</string>
```

Comment:
This is a requirement from stakeholders  Eva and the MOH team brought it up.

Reply:
ok, but it is not implemented yet so if you change label now it will be confusing that user can search by mother name but in reality he can't because it is not implemented